docs/api/permissions.rst

```diff
@@ -0,0 +1,332 @@
++----------------+------------------------+----------------------------------+
+| Bucket         | ``write``              | Ability to write + read on the   |
+|                |                        | bucket and all children objects. |
+|                +------------------------+----------------------------------+
+|                | ``read``               | Ability to read all objects in   |
+|                |                        | the bucket.                      |
+|                +------------------------+----------------------------------+
+|                | ``collections:create`` | Ability to create new            |
+|                |                        | collections in the bucket.       |
+|                +------------------------+----------------------------------+
+|                | ``groups:create``      | Ability to create new groups     |
+|                |                        | in the bucket.                   |
++----------------+------------------------+----------------------------------+
+| Collection     | ``write``              | Ability to write and read all    |
+|                |                        | objects in the collection.       |
+|                +------------------------+----------------------------------+
+|                | ``read``               | Ability to read all objects in   |
+|                |                        | the collection.                  |
+|                +------------------------+----------------------------------+
+|                | ``records:create``     | Ability to create new records    |
+|                |                        | in the collection.               |
++----------------+------------------------+----------------------------------+
+| Record         | ``write``              | Ability to write and read the    |
+|                |                        | record.                          |
+|                +------------------------+----------------------------------+
+|                | ``read``               | Ability to read the record.      |
+|                |                        |                                  |
++----------------+------------------------+----------------------------------+
+| Group          | ``write``              | Ability to write and read the    |
+|                |                        | group.                           |
+|                +------------------------+----------------------------------+
+|                | ``read``               | Ability to read the group.       |
+|                |                        |                                  |
++----------------+------------------------+----------------------------------+
+
+
+By default the ``write`` permission is given to the creator of an
+object.
+
+
+.. note::
+
+  There is no ``delete`` permission: Anyone with the ``write`` permission on an
+  object can also edit its associated permissions and delete it.
```

Comment:
I replaced with "There is no ``delete`` permission. Anyone with the ``write`` permission on an
  object can delete it."